### DISCUSSION TITLE
Nuxt - images load slowly and shifts the whole layout

### DISCUSSION THREAD

**Discussion by emsitkowski** [nuxt/nuxt#8631] [Questions (Nuxt 2)]
I have problem with my Nuxt simple website. The images on page load doesn't load fast enough. It makes all the layout shift and for a while I can only see their alt text. Why nuxt doesn't wait for all the resources? It just downloads like half of the page size and later lazy load the rest.

**Comment by devlsh** [comment]
Make sure you specify `width` and `height` on the `<img>` element for layout shifts to be avoided

**Reply by emsitkowski** [reply]
What about responsive layout with width: 100% etc?
